cmd/start.go

```diff
@@ -103,12 +114,18 @@ func start(cmd *cobra.Command) error {
 
 	scheduler := generator.StartScheduler()
 
+	// TODO: Replace with a `isBootstrapNode` config flag.
+	// Monitor sortition pool only if the node is a non-bootstrap node, i.e.
+	// it has peers that it should connect to.
```

Comment:
A bootstrap node will also have the peers that it will connect to (other bootstrap nodes).
Maybe we could add a `--network.bootstrap` boolean flag to the client start command?

Reply:
See https://github.com/keep-network/keep-core/pull/3256/commits/d13453e766bce453b6fb1939b36b2555c0b144e7